chat-components/src/components/chatbutton/common/defaultStyles/defaultChatButtonSubTitleStyles.ts

```diff
@@ -12,5 +12,6 @@ export const defaultChatButtonSubTitleStyles: IStyle = {
     overflow: "hidden",
     padding: "0px",
     textOverflow: "ellipsis !important",
-    width: "max-content"
+    whiteSpace: "nowrap",
```

Comment:
Please remember to update dummyDefaultProps.ts after uptaking the chat-component version in chat-widget